src/binary/mod.rs

```diff
@@ -29,3 +29,22 @@ pub fn decode(ids: &walrus::IndicesToIds, from: &[u8]) -> Result<WebidlBindings,
     WebidlBindings::decode(&mut cx, &mut from)?;
     Ok(cx.webidl_bindings)
 }
+
+/// Callback for `walrus::ModuleConfig::on_parse` to parse thea webidl bindings
+/// custom section if one is found.
+pub fn on_parse(
+    module: &mut walrus::Module,
+    ids: &walrus::IndicesToIds,
+) -> Result<(), failure::Error> {
+    let section = module
+        .customs
+        .iter()
+        .find(|(_, section)| section.name() == "webidl-bindings");
```

Comment:
This should be `module.customs.remove_raw("webidl-bindings")` no? You don't want to leave the raw custom module in and also have the trait object in there.

Reply:
Oops, you are correct!